### PR TITLE
Completed css-two-column-layout exercise. - Justin Son

### PR DESCRIPTION
**Pull request by justintaeson**
css-two-column-layout_1 31 22

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Justin, your output is missing the button, please revise

**Comment by justintaeson** [comment]
css-two-column-layout_V2_2 1 22

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  width: 1024px;
+  height: 768px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:11]: `column` what? Remember that column classes should be named after their size.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by justintaeson** [comment]
css-two-column-layout_v2_2 6 22

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**